tools/extra/fpgadiag/mactest.py

```diff
@@ -31,11 +31,11 @@
 import binascii
 import sys
 import glob
-from common import FpgaFinder, exception_quit
-from common import COMMON, convert_argument_str2hex
+from common import FpgaFinder, exception_quit, COMMON, hexint
+
+SYSF_IF = '/sys/class/net'
+DIVIDE = '-' * 80
 
-sys_if = '/sys/class/net'
-divide = '-' * 80
 FVL_SIDE = 1
```

Comment:
Should we get rid of code names? Maybe this can be HOST_SIDE?

Reply:
Fixed